server/lib/notifications.js

```diff
@@ -236,9 +238,68 @@ const notifyUpdateSubscribers = async activity => {
   const emailOpts = { from: activity.data.fromEmail };
   const update = await models.Update.findByPk(activity.data.update.id);
   const allUsers = await update.getUsersToNotify();
-  return notifySubscribers(allUsers, activity, emailOpts);
+  const modifiedActivity = replaceVideosByImagePreviews(activity);
+  return notifySubscribers(allUsers, modifiedActivity, emailOpts);
 };
 
+function replaceVideosByImagePreviews(activity) {
+  sanitizerOptions.transformTags = {
```

Comment:
Doing so is mutating the `sanitizerOptions` from `../models/Update`.

I suggest to do something like:

```es6
import { sanitizerOptions as updateSanitizerOptions } from '../models/Update';

function replaceVideosByImagePreviews(activity) {
  const sanitizerOptions = {
    ...updateSanitizerOptions,
    ...
  }
```

Reply:
Done. Thanks for pointing it out. 👍🏽